### PR TITLE
Add Initial Aruba ClearPass Report

### PR DESCRIPTION
**Pull request by alagoutte**
with System, Licence, Network Device (and Group) and Service (Enforcement Profil and Profiles)

### CODE REVIEW

File: Src/Private/Get-AbrCPPMNetworkDevice.ps1 [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    param (
+
+    )
+
+    begin {
+        Write-PScriboMessage "Discovering Network Device settings information from $System."
+    }
+
+    process {
+        PageBreak
```
Comment (tpcarman) [inline Src/Private/Get-AbrCPPMNetworkDevice.ps1]: Sections should immediately follow the previous section. Is there a particular reason a pagebreak is required?

File: Src/Private/Get-AbrCPPMService.ps1 [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    param (
+
+    )
+
+    begin {
+        Write-PScriboMessage "Discovering Service settings information from $System."
+    }
+
+    process {
+        PageBreak
```
Comment (tpcarman) [inline Src/Private/Get-AbrCPPMService.ps1]: Sections should immediately follow the previous section. Is there a particular reason a pagebreak is required?

**Review by tpcarman — CHANGES_REQUESTED** [review]

File: Src/Private/Get-AbrCPPMNetworkDevice.ps1 (hunk 1 above)
Comment (alagoutte) [reply Src/Private/Get-AbrCPPMNetworkDevice.ps1]: it is only better display for me a new chapiter start of top of page but i can remove if needed

Comment (tpcarman) [reply Src/Private/Get-AbrCPPMNetworkDevice.ps1]: I understand your reasoning however I'd also like reports to remain consistent in their outlay. I think sections should immediately follow the previous section.

**Comment by alagoutte** [comment]
@tpcarman ready to merge ! (i will make a new PR for README and sample)

**Review by tpcarman — APPROVED** [review]

**Comment by alagoutte** [comment]
@tpcarman do you have an idea why the lint fail ?

**Pull request merged**